security/providers/jwt/src/main/java/io/helidon/security/providers/jwt/JwtProvider.java

```diff
@@ -603,7 +604,7 @@ public Builder validitySeconds(long validitySeconds) {
                 .tokenHeader("Authorization")
                 .tokenPrefix("bearer ")
                 .build();
-        private OutboundConfig outboundConfig;
+        private OutboundConfig outboundConfig = OutboundConfig.builder().build();
```

Comment:
Other defaults are defined in the builder, wouldn't it be a better place for this one? 
https://github.com/oracle/helidon/blob/ce28e044a5e13428a6d16f946b2fe10709fd6ed4/security/providers/jwt/src/main/java/io/helidon/security/providers/jwt/JwtProvider.java#L594-L611

Reply:
It is, sorry my bad